app/views/jobseekers/job_applications/professional_body_memberships/new.html.slim

```diff
@@ -18,4 +18,4 @@
       = f.govuk_submit t("buttons.save_and_continue")
 
     .govuk-button-group
-      = govuk_link_to t("buttons.cancel"), jobseekers_job_application_build_path(job_application, :professional_body_membership)
+      = govuk_link_to t("buttons.cancel"), jobseekers_job_application_build_path(job_application, :professional_body_memberships)
```

Comment:
I would normally always add a test when fixing such defects - they show that our testing is lacking in certain places